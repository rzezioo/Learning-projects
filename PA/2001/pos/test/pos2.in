10 34
1
1
1
1
1
1
1
1
1
1
7 7
4 10
2 7
1 3
5 4
10 4
3 1
3 8
8 7
4 5
7 8
4 3
5 1
5 2
6 7
7 4
9 9
8 9
8 8
4 4
8 3
7 6
6 1
2 2
3 2
5 2
9 6
1 1
2 5
3 10
10 8
1 5
1 5
2 3
