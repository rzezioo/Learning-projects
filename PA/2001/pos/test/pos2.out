34
1 3 1 5 4 10 4 5 2 2 7 7 8 3 4 4 7 6 7 8 8 3 2 5 2 3 10 8 9 9 6 1 1 5 1